data-prepper-core/src/main/java/com/amazon/dataprepper/parser/config/DataPrepperAppConfiguration.java

```diff
@@ -55,4 +57,9 @@ public DataPrepperConfiguration dataPrepperConfiguration(
             return new DataPrepperConfiguration();
         }
     }
+
+    @Bean
+    public Optional<PluginModel> pluginModel(final DataPrepperConfiguration dataPrepperConfiguration) {
```

Comment:
This value is the authentication. So this method should be renamed to `authentication` from `pluginModel`.